changelog/fragments/sdk-fbc-run-bundle(-upgrade).yaml

```
@@ -0,0 +1,19 @@
+entries:
+  - description: >
+      Add File-Based Catalog support for both `run bundle` and `run bundle-upgrade` subcommands. 
+      Infer the image type and handle FBC scenarios accordingly for both commands. For FBC images,
+      Operator Registry APIs are called for rendering the bundle and/or the index image and serving these images
+      through the registry pod using a gRPC port. For images that are SQLite based, 
+      `opm registry add` and `opm serve` commands will be called during the registry pod creation. 
+      Support for these old opm commands will be deprecated and completely removed in the future. 
+
+    # kind is one of:
+    # - addition
+    # - change
+    # - deprecation
+    # - removal
+    # - bugfix
+    kind: addition
```

```suggestion
    kind: change
```
it's a change to an existing command. Though I'm not entirely sure the distinction between change, addition and removal because addition and removal are also "changes" :)